Cargo.toml

```diff
@@ -6,28 +6,28 @@ edition = "2021"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-async-broadcast = "0.7.0"
-clap = { version = "4.4", features = ["derive", "env"] }
-async-std = { version = "1.9.0", features = ["unstable", "attributes"] }
-async-compatibility-layer = { git = "https://github.com/EspressoSystems/async-compatibility-layer.git", tag = "1.4.2", features = [
+async-compatibility-layer = { git = "https://github.com/EspressoSystems/async-compatibility-layer.git", tag = "1.4.1", features = [
```

Comment:
Um... so this is unfortunate, we're stuck either broken with respect to hotshot, or broken with respect to sequencer, and both of them will be resolved soon, since this can be replaced with
```toml
async-compatibility-layer = { version = "1.4", features = [
```
but we need the dependencies to do the same.